apmpackage/apm/0.1.0/data_stream/internal_metrics/manifest.yml

```diff
@@ -1,4 +1,3 @@
 title: APM internal metrics
 type: metrics
 dataset: apm.internal
-ingest_pipeline: apm
```

Comment:
Why have these been removed?

Reply:
Because I am renaming the pipelines from `apm.json` to `default.json`, that don't need this. I am doing this because of a bug in Kibana that will cause to duplicate `apm.json` as `default.json`. Didn't file it yet, but I'll do it today.

Reply:
OK cool, thanks!